src/Classes/Domain/Repository/DepositLicenseLogRepository.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace EWW\Dpf\Domain\Repository;
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+/**
+ * The repository for DepositLicenseLog
+ */
+class DepositLicenseLogRepository extends \EWW\Dpf\Domain\Repository\AbstractRepository
+{
```

Comment:
I don't know if there is a way to avoid classes like that. I always assumed we need for each database model we need a repository class, even if we only use standard functionality.